quartzlib/pom.xml

```diff
@@ -110,6 +110,17 @@
             <version>1.1.1</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.graalvm.js</groupId>
+            <artifactId>js</artifactId>
+            <version>20.3.0</version>
+        </dependency>
+        <dependency>
```

Comment:
Inconsistency in the line feed, there is no spacing for bukkit dependency and jetbrains but there are spacing between junit and junit-jupiter